sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableDropConstraintParseSuite.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.execution.command
+
+import org.apache.spark.sql.catalyst.analysis.{AnalysisTest, UnresolvedTable}
+import org.apache.spark.sql.catalyst.parser.CatalystSqlParser.parsePlan
+import org.apache.spark.sql.catalyst.parser.ParseException
+import org.apache.spark.sql.catalyst.plans.logical.DropConstraint
+import org.apache.spark.sql.test.SharedSparkSession
+
+class AlterTableDropConstraintParseSuite extends AnalysisTest with SharedSparkSession {
```

Comment:
This is for `DropConstraint`. Is there another test suite for `AddConstraint` too? Seems I don't find it.

Reply:
It is in the other test suites, such as `CheckConstraintParseSuite`/`PrimaryKeyConstraintParseSuite`, etc